po/tr.po

```diff
@@ -1738,23 +1754,23 @@ msgstr ""
 
 #: rust/libnewsboat/src/configpaths.rs:149
 msgid "Migrating configs and data from Newsbeuter's XDG dirs..."
-msgstr ""
+msgstr "Yapılandırmalar ve veriler Newsbeuter'in XDG dizinlerinden alınıyor..."
 
 #: rust/libnewsboat/src/configpaths.rs:205
 msgid "Migrating configs and data from ~/.newsbeuter/..."
-msgstr ""
+msgstr "Yapılandırmalar ve veriler ~/.newsbeuter/'den alınıyor..."
 
 #: rust/libnewsboat/src/configpaths.rs:217
 msgid "Aborting migration because mkdir on `%s' failed: %s"
 msgstr "Göç iptal ediliyor çünkü `%s' üzerinde mkdir başarısız oldu: %s"
 
 #: rust/regex-rs/src/lib.rs:158 rust/regex-rs/src/lib.rs:163
 msgid "regcomp returned code %i"
-msgstr ""
+msgstr "regcomp %i kodunu döndürdü"
 
 #: rust/regex-rs/src/lib.rs:243 rust/regex-rs/src/lib.rs:247
 msgid "regexec returned code %i"
-msgstr ""
+msgstr "regexec %i kodunu döndürdü"
 
 #~ msgid "config"
```

Comment:
Please remove those, these are just leftovers of previous edits.

Reply:
I am not sure what to remove, the whole string?

Reply:
I mean the lines that start with `#~`.